python/pyspark/pandas/window.py

```diff
@@ -2448,11 +2449,26 @@ def mean(self) -> FrameLike:
         unified_alpha = self._compute_unified_alpha()
 
         def mean(scol: Column) -> Column:
-            sql_utils = SparkContext._active_spark_context._jvm.PythonSQLUtils
+            if is_remote():
+                from pyspark.sql.connect.functions import _invoke_function_over_columns, lit
+
+                col_ewm = _invoke_function_over_columns(
+                    "ewm",
```

Comment:
nit: I guess we can name it a bit more ps-specific `pandas_ewm`

Reply:
Oh, I just checked the comments. Will update them as a follow-up PRs.

Reply:
oh, no problem, on second thought, I think this naming and PR is fine enough

Reply:
Got it. Thanks for the review! :-)